### ISSUE TITLE
RKE provisioned and imported cluster removing namespace leaves k8s in terminating/removing state

### ISSUE THREAD

**Issue by kodo651** [rancher/rancher#22189]
Hi!
To me it looks like this issue (#14715) hasn't been solved. I'm running a PoC cluster in vSphere (v6.7) - one etcd/controlplane host and four worker nodes. The cluster runs k8s v1.14.3, Rancher v2.2.1 and Docker v18.9.2.

If I create a namespace in a RancherOS based and Rancher provisioned cluster everything works fine. However, if I use RKE to provision my PoC cluster based on CentOS7 (1810), imports the cluster into Rancher and *then* try to do the same, k8s "seems to hang forever" in terminating state and the Rancher UI says "Removing". To me it sounds like the issues reported earlier with one exception. Although I remove the lines (kubectl edit ns <namespace>)
```
  finalizers:
  - controller.cattle.io/namespace-auth
```
and then issue `kubectl delete ns <namespace>`, it says namespace "<namespace> deleted" and then hangs "forever". The Rancher UI says "Removing". As my setup is extremely basic with NO workload deployed whatsoever, I would assume there's a bug lurking in the combination of RKE provisioned and imported k8s clusters into Rancher. I'm not super knowledgeable about the inner workings of k8s and Rancher so I'm a "little bit stuck right now"...

(a more descriptive error message from Rancher UI):
```
{

    "baseType": "error",
    "code": "Conflict",
    "message": "Operation cannot be fulfilled on namespaces \"<namespace>\": The system is ensuring all content is removed from this namespace.  Upon completion, this namespace will automatically be purged by the system.",
    "status": 409,
    "type": "error"

}
```

_Originally posted by @kodo651 in https://github.com/rancher/rancher/issues/14715#issuecomment-521555115_

**Comment by vprocopan** [comment]
same here!
![image](https://user-images.githubusercontent.com/6999055/100077383-f6eb3780-2e4a-11eb-8a1b-44af1ebf3587.png)

**Comment by C-PET** [comment]
Seem to have same issue. creating an empty namespace using Rancher UI
once created turning right around and attempting to delete using rancher UI leaves it in terminating state.

my annotations have no controller.cattle.io/namespace-auth
I only have lifecycle.cattle.io/create.namespace-auth : true.

I have tried multiple different solutions suggested
from editing the output of json and then kubectl replace --raw ...
just returns error
Error from server (Conflict): Operation cannot be fulfilled on namespaces "testnamespace": StorageError: Invalid object, Code: 4, Key:/registry/namespaces/testnamespaces, ResourceVersion: 0, AdditionalErrorMsg: Precondition failed: UID in precondition: 1037187a-4339-4540-b660-d8c4307e1a6f, UID in object meta:

if I search for resources in the namespace that might still be running, returns nothing.

kubectl api-resources --verbs=list -namespaced -o name | xargs -n 1 kubectl get --show-kind --ignore-not-found -n testnamespace

returns nothing

Rancher 2.4.5
Kube : 1.18.3

**Comment by C-PET** [comment]
So I did eventually solve my issue.  

Rebooting the controller.etcd nodes that control the cluster I have solved the issue.  Apparently an API-Service was not responding to requests which caused the delete request to never complete.   After rebooting the active etcd,controlplane node they simply disappeared.   Likely just a hiccup/corruption in the etcd node.

We have 3 controlplane, etcd nodes that control our downstream K8S cluster from rancher.  Rebooted them one by one and it cleared all "terminating" namespaces
